install/installation-kubernetes.md

```diff
@@ -10,27 +10,18 @@ Before you begin installing TimescaleDB on a Kubernetes deployment, make sure yo
 * [kubectl][kubectl-install]
 * [Helm][helm-install]
 * [Kubernetes Cluster][kubernetes-install]
+* (**Optional**) Create a YAML file that specifies the values for the parameters that are provided while installing the chart. For example, you can create a new `myvalues.yaml` file. For details about the parameters you can set, see  the [Administrator Guide][admin-guide].
 
 </highlight>
 
-## Install the TimescaleDB using Helm chart
-Before you install the TimescaleDB Helm chart, you need to configure these
-settings in the `values.yaml` configuration file:
-*   Credentials for the superuser, admin, and other users
-*   TLS Certificates
-*   **Optional:** `pgbackrest` [configuration][timescale-backups]
-
-<highlight type="note">
-If you do not configure the user credentials before you start, they are randomly
-generated. When this happens, the `helm upgrade` command does not rotate the
-credentials, to prevent breaking the database by changing the database
-credentials instead it uses the same credentials that are generated during the
-`helm install`.
-</highlight>
+## Install TimescaleDB using a Helm chart
+You can install TimescaleDB on Kubernetes using a Helm chart with the default `values.yaml`. The user credentials are randomly generated during the installation with the default `values.yaml` file.
+When you use the `helm upgrade` command it does not rotate the credentials, to prevent breaking the database by changing the database credentials instead it uses the same credentials that are generated during the `helm install`.
```

Comment:
```suggestion
When you then use the `helm upgrade` command, it does not rotate the credentials, because changing the database credentials would break the database. Instead, it continues to use the credentials generated during `helm install`.
```